pkg/clusters/addons/knative/builder.go

```diff
@@ -1,31 +1,32 @@
 package knative
 
 import (
-	"errors"
+    "errors"
```

Comment:
These spaces don't look right. Should be tabs for *.go files

Reply:
For some reason, my IDE (GoLand) has detected this project uses spaces 🤔 Sorry for that.